index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="EN">
+  <head>
+    <meta charset="utf-8">
+    <title>My Portfolio</title>
+    <meta name="description" content="">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;500;600;700;800;900&display=swap" rel="stylesheet"> 
+    <link rel="stylesheet" href="css/reset.css">
+    <link rel="stylesheet" href="css/styles.css">
+  </head>
+  <body>
+    <header class="header">
+      <h2>Welcome</h2>
+      <a title="Button"></a>
+    </header>
+    <section class="headline">
+      <h1>Hey there. I'm Amaury</h1>
+      <h1>I'm a software developer</h1>
```

Comment:
[REQUIRED] Please use only one h1 element on your page as this should represent the main heading/subject for the whole page.